### PR TITLE
Including icons in the plugin panel

### PR DESCRIPTION
**Pull request by Poffen**
I've included icons for the toggle function in the plugins panel, and also enclosed the toggle footnote in a div container. 

The icons are modified (reduced size) from professional icons made by Icojam (http://www.icojam.com/?s=Blueberry+Basic) and should be free of any known copyrights.

Putte

### CODE REVIEW

File: featherbb/View/admin/plugins.php [hunk 1]
```diff
@@ -58,7 +59,7 @@
 <?php endforeach; ?>
                                 </tbody>
                             </table>
-                            <tfoot><strong>* <?php _e('Toggle active state'); ?></strong></tfoot>
+                            <tfoot><div id="toggle">* <?php _e('Toggle active state'); ?></div></tfoot>
```
Comment (beaver-dev) [inline featherbb/View/admin/plugins.php]: Could you place the tfoot tag one line above, just after the tbody of the table tag ? I placed it wrong.
And change the div id to "toggle-description". Yes, I'm a maniac :P

File: style/themes/FeatherBB/base_admin.css [hunk 2]
```diff
@@ -182,3 +182,6 @@ input[type=radio]:focus {
 #adminconsole #upload-plugin div {
     display: inline;
 }
+#toggle {
```
Comment (beaver-dev) [inline style/themes/FeatherBB/base_admin.css]: Could you change the rule to `#adminconsole #toogle-description {}` ?

**Comment by beaver-dev** [comment]
Hi Putte,
I made two minor suggestions, your PR will be accepted as soon as you made the changes.

**Comment by Poffen** [comment]
Hi beaver-dev,

It sounds as if you are a meticulous man, and that's a good thing! 

Yes the tfoot tag should of course be nested inside the table tag, but it's probably not the solution in this case. Accordng to [w3schools](http://www.w3schools.com/tags/tag_tfoot.asp), the tfoot tag should be placed above the tbody tag. That means that the footnote will appear above the table content, which is probably not what you want? So my suggestion is to remove the tfoot tag and place the toggle div container after the table. 

I would also suggest a few more minor css adjustments: 
- It seems that the content of columns is jumping around when there is no content in the action-column (when all plugins are activated). To prevent this I suggest to fix the width of the action column by applying a min-widt of 75 px to the plugin-actions class.
- I also added a 10 px padding (left) to the toggle div in order to align the toggle text with text of the table.
- I think it would be nice to separate the toggle text from the table by applying a bottom border (same color and thickness) to the td cells of the table (see the screenshots I attached in the forum post). I can use `#adminconsole td` to restrict it to the admin part, but it may of course affect other tables in the admin area since there seems to be no way to restrict this to the plugin panel. What do you think?
- Lastly, I applied a class to the icons in the first place, which I forgot to include in the css file. I suggest to include this as `#adminconsole img.pluginIcons`.

Let me know what you think about these adjustments, and then I'll push it. 

Putte

**Comment by Poffen** [comment]
I think I was a little quick on the tfoot issue... If you place the tfoot tag where it should be (before the tbody tag), the content will surprisingly end up at the bottom of the table, which of course will solve the issue. The tfoot tag should contain tr and td tags, but you can use a single td tag and span (colspan) it over all columns to stretch the text over the entire row. This way you also get a nice separation of the toggle text and the table content above (because the toggle text is inside a td tag). So this should be feasible...

Putte

**Comment by Poffen** [comment]
Hi again,

I'm not sure I did the right thing here, but I pushed another change which was amended to the current PR. It should perhaps have been filed as a separate request. 

Anyway, the `<?= Url::base_static(); ?>` points to the root of the server and blocks styling of the installation view when FeatherBB is installed in a sub directory. I suggest changing this to `<?= Url::base() ?>` instead.

Putte

**Comment by beaver-dev** [comment]
It seems good to me! PR merged.
Concerning the CSS, don't spend too mutch time on it : after implementing a new permissions and preferences system we will rewrite theme using a SASS version of http://getskeleton.com/, where we can use responsive columns among other wonderful and beautiful design patterns.

**Pull request merged**